diesel/src/migrations/mod.rs

```diff
@@ -163,7 +163,7 @@ pub fn setup_database<Conn: Connection>(conn: &Conn) -> QueryResult<usize> {
 fn create_schema_migrations_table_if_needed<Conn: Connection>(conn: &Conn) -> QueryResult<usize> {
     conn.silence_notices(|| {
         conn.execute("CREATE TABLE IF NOT EXISTS __diesel_schema_migrations (
-            version VARCHAR PRIMARY KEY NOT NULL,
+            version VARCHAR(255) PRIMARY KEY NOT NULL,
```

Comment:
Nah, just have a test and a noncryptic error message. Maybe. Later. It's not critical.